### ISSUE TITLE
Failed to activate package - Cannot read property 'getGrammar' of undefined

### ISSUE THREAD

**Issue by mupchrch** [cakecatz/flex-toolbar#75]
Happens when I load Atom and there is no active editor.

**Atom Version**: 1.8.0
**System**: Mac OS X 10.10.5
**Thrown From**: [flex-tool-bar](https://github.com/cakecatz/flex-toolbar) package, v0.10.1

### Stack Trace

Failed to activate the flex-tool-bar package

```
At Cannot read property 'getGrammar' of undefined

TypeError: Cannot read property 'getGrammar' of undefined
    at Object.module.exports.storeGrammar (/Users/upchm/.atom/packages/flex-tool-bar/lib/flex-tool-bar.coffee:289:29)
    at Object.module.exports.activate (/Users/upchm/.atom/packages/flex-tool-bar/lib/flex-tool-bar.coffee:36:6)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:183:20)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:156:32
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:92:15)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:149:26
    at Package.module.exports.Package.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:146:34)
    at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:538:21)
    at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/package-manager.js:452:29
    at exit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/package-manager.js:73:16)
    at triggerExitCallback (/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:215:47)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:222:18
    at Socket.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:100:18)
    at emitOne (events.js:82:20)
    at Socket.emit (events.js:169:7)
    at Pipe._onclose (net.js:469:12)
```

### Commands

```
     -0:40.5.0 editor:delete-to-beginning-of-line (atom-text-editor.editor.mini.is-focused)
     -0:34.8.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:09.8.0 window:toggle-dev-tools (atom-workspace.workspace.scrollbars-visible-always.seti-ui-no-icons.compact-tab-bar.panel-contrast.use-animations.compact-tree-view.theme-firewatch-syntax.theme-atom-material-ui.has-custom-icons)
```

### Config

```json
{
  "core": {
    "allowPendingPaneItems": false,
    "openEmptyEditorOnStart": false,
    "packagesWithKeymapsDisabled": [
      "activate-power-mode"
    ],
    "themes": [
      "atom-material-ui",
      "firewatch-syntax"
    ]
  },
  "flex-tool-bar": {
    "showConfigButton": false
  }
}
```

### Installed Packages

```coffee
# User
activate-power-mode, v0.5.2
atom-material-ui, v1.2.13
atom-typescript, v8.11.0
docblockr, v0.7.3
file-icons, v1.7.14
firewatch-syntax, v1.0.1
flex-tool-bar, v0.10.1
glowing-cursor, v0.1.1
highlight-selected, v0.11.2
linter, v1.11.10
minimap, v4.24.6
minimap-find-and-replace, v4.5.1
minimap-highlight-selected, v4.4.0
minimap-linter, v1.1.1
minimap-selection, v4.4.0
minimap-split-diff, v0.3.1
outline-selection, v0.2.1
pigments, v0.29.2
tool-bar, v0.4.0
tree-view-filter, v0.4.1

# Dev
No dev packages
```

**Comment by mupchrch** [comment]
Seems to have to do with [this line](https://github.com/cakecatz/flex-toolbar/blob/master/lib/flex-tool-bar.coffee#L288). getActiveTextEditor() is returning undefined because I didn't have an editor open when I loaded Atom - I had settings view open.

**Comment by markvandenbrink** [comment]
I have the same problem after updating the toolbar to 0.10.1. Reverted back to 0.10.0 which works fine. Hopefully just a small bug which can be fixed easily.

**Comment by areel** [comment]
The error being reported within Atom is https://github.com/cakecatz/flex-toolbar/issues/25
Doing as @mupchrch states worked for me.

**Comment by markvandenbrink** [comment]
Looks like it is fixed in 0.10.2! Thanks!

**Comment by mupchrch** [comment]
Thanks for the quick fix :)
